modules/ROOT/pages/manage-exchange-api-task.adoc

```diff
@@ -7,45 +7,41 @@ You need to have access to an API in your organization's Exchange to perform thi
 
 This procedure covers the configuration of the API. After saving the configuration, you can deploy the API.
 
-. Click API Manager in tne Anypoint Platform navigation bar.
-+
-API Administration opens in your default environment, such as Sandbox.
-+
-. In API Administration, select Manage API > Manage API From Exchange.
+. In API Manager, select *Manage API* > *Manage API From Exchange*.
 +
 image::environment.png[]
 +
-. In Manage API from Exchange - API Configurations, select options:
-+
-* In API Name, type the first characters of the API name, and then API Manager displays APIs to choose from. Select the API you want to manage from the list of available APIs in Exchange. In API Version, select a version of the API. In Asset Version, select the Exchange asset version.
+. In *API Name*, type the first few characters of the API name, and click the search icon.
+. Select the API that you want to manage from the list of available APIs in Exchange. 
 +
-A link to the RAML/OAS specification for viewing appears.
+The *Asset type*, *API version*, and *Asset version* fields are auto-populated.   
 +
-* In Managing Type, choose Basic Endpoint or Endpoint with Proxy.
+NOTE: OAS 3.0 specifications are currently not supported in API Manager. You must convert the API to RAML or OAS 2.0 to manage it using API Manager.
 +
-Set up a basic endpoint if you create your API outside API Manager, but you want to apply policies or otherwise manage the API using API Manager. For example, if you create the API using Mule Runtime, you don't need a proxy.
+. In *Managing Type*, choose the type of API endpoint that you want to create:
+* Basic Endpoint: Use this option if you created your API outside API Manager, but want to apply policies or otherwise manage the API using API Manager. For example, if you created the API using Mule Runtime, you do not need a proxy. 
+* Endpoint with Proxy: Use this option if you want to provide additional security for your API. 
```

Comment:
GLOBALLY change all bullet-list items to break after the name or first line.
If you have a name, you might consider using a definition list; see style reference.
Note that these are UI labels and so should be in bold; otherwise, I would not capitalize "basic endpoint" or any other endpoint.

```suggestion
. In *Managing Type*, choose the type of API endpoint that you want to create:
* *Basic Endpoint* +
Use this option if you created your API outside of API Manager but want to apply policies or otherwise manage the API using API Manager. For example, if you created the API using Mule runtime engine (Mule), you do not need a proxy. 
* *Endpoint with Proxy* +
Use this option if you want to provide additional security for your API. 
```